browser/src/Input/Mouse.ts

```diff
@@ -61,7 +61,10 @@ export class Mouse extends EventEmitter {
                 } else {
                     scrollcmdY += `ScrollWheelDown>`
                 }
-                this.emit("mouse", scrollcmdY)
+                this.emit("mouse",scrollcmdY + `<${line},${column}>`)
```

Comment:
Can you please add a space after the comma here? I'm surprised tslint didn't complain about it!